.lintstagedrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Changed the file format to a simpler one.

The `.js` format needed us to create the whole command which is error-prone (eg: try creating and committing a file with spaces in the name). The next.js documentation does not consider such file names for some reason.

One change is that we now use plain `eslint` over `next lint` which is fine because the latter just adds some sugar which we don't need: https://github.com/vercel/next.js/issues/33096#issuecomment-1007920000